components/admin_console/system_users/index.js

```diff
@@ -20,10 +24,32 @@ function mapStateToProps(state) {
     const enableUserAccessTokens = config.EnableUserAccessTokens === 'true';
     const experimentalEnableAuthenticationTransfer = config.ExperimentalEnableAuthenticationTransfer === 'true';
 
+    const search = state.views.search.systemUsersSearch;
+    let totalUsers = 0;
+    let searchTerm = '';
+    let teamId = '';
+    if (search) {
+        searchTerm = search.term || '';
+        teamId = search.team || '';
+
+        if (!teamId || teamId === SearchUserTeamFilter.ALL_USERS) {
+            const stats = state.entities.admin.analytics || {[Stats.TOTAL_USERS]: 0, [Stats.TOTAL_INACTIVE_USERS]: 0};
```

Comment:
This should be done through a selector so that it doesn't create a new object each time when `state.entities.admin.analytics` is empty. Perhaps you could just make a `getTotalUsers` selector for this page to obscure that logic

Reply:
That object isn't getting passed in, I'm passing in `totalUsers` as an integer